scripts/pre-push

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+//requirements
+const ethereumjs = require('ethereumjs-util'),
+	https = require('https'),
+	exec = require('child_process').exec,
+	fs = require('fs');
+
+//constants
+const host = 'https://api.etherscan.io/api?module=account&action=balancemulti&address=',
+	apiKey = '75CG3FABT3B8UNKXKJZJMRZUZKN1VQW5GU';

```

Comment:
There probabably isn't a need to commit an API key. 5 requests/second limit should be OK for us: https://ethereum.stackexchange.com/questions/34190/does-etherscan-require-the-use-of-an-api-key. Maybe we can split the requests and wait for 1 second between batches of 5 addresses? Otherwise, we run into the risk that someone steals your quota by grabbing your API key anyway.